src/act.obj.cpp

```diff
@@ -45,7 +45,7 @@ void calc_weight(struct char_data *ch);
 SPECIAL(weapon_dominator);
 
 int wear_bitvectors[] = {
-                          ITEM_WEAR_TAKE, ITEM_WEAR_HEAD, ITEM_WEAR_EYES, ITEM_WEAR_EAR,
+                          ITEM_WEAR_TAKE, ITEM_WEAR_HEAD, ITEM_WEAR_EYES, ITEM_WEAR_EAR, ITEM_WEAR_MOUTH,
```

Comment:
This new value needs to be at the end of the wear_bitvectors[] array, after wear_feet. This ensures that we don't mess up any existing wear locations, and also that your string for wearing the item (which you set below) is in the right spot.

Reply:
I think I did it right this time?